google/cloud/testing_util/opentelemetry_matchers.h

```diff
@@ -69,6 +69,12 @@ MATCHER_P(EventAttributesImpl, matcher,
                                        result_listener);
 }
 
+MATCHER_P(LinkAttributesImpl, matcher,
```

Comment:
No action needed....

These AttributeImpl matchers can be combined if you know how to use templates. Sadly, I cannot help.

Reply:
Yea.... I don't understand it either....